examples/v1beta1/README.md

```diff
@@ -1,391 +1,141 @@
+
+Check the [Hyperparameter Tuning](https://www.kubeflow.org/docs/components/katib/overview/#hyperparameters-and-hyperparameter-tuning)
+Experiments for the following algorithms:
+
+- [Random Search](./hp-tuning/random.yaml)
+
+- [Grid Search](./hp-tuning/grid.yaml)
+
+- [Bayesian Optimization](./hp-tuning/bayesian-optimization.yaml)
+
+- [Tree of Parzen Estimators (TPE)](./hp-tuning/tpe.yaml)
+
+- [Multivariate TPE](./hp-tuning/multivariate-tpe.yaml)
+
+- [Covariance Matrix Adaptation Evaluation Strategy (CMA-ES)](./hp-tuning/cma-es.yaml)
+
+- [Sobol's Quasirandom Sequence](./hp-tuning/sobol.yaml)
+
+- [HyperBand](./hp-tuning/hyperband.yaml)
+
+### Neural Architecture Search
+
+Check the [Neural Architecture Search](https://www.kubeflow.org/docs/components/katib/overview/#neural-architecture-search)
+Experiments for the following algorithms:
+
+- [Efficient Neural Architecture Search (ENAS)](./nas/enas-gpu.yaml)
+
+- [Differentiable Architecture Search (DARTS)](./nas/darts-gpu.yaml)
+
+### Early Stopping
+
+Improve your Hyperparameter Tuning Experiments with the following
+[Early Stopping](https://www.kubeflow.org/docs/components/katib/early-stopping/) algorithms:
+
+- [Median Stopping Rule](./early-stopping/median-stop.yaml)
+
+## Katib Python SDK Examples
+
+To learn more about Katib Python SDK check [this directory](./sdk).
+
+## Resume Katib Experiments
+
+You can use different resume policies in Katib Experiments. Follow
+[this guide](https://www.kubeflow.org/docs/components/katib/resume-experiment/)
+to know more about it. Check the following examples:
+
+- [Resume From Volume](./resume-experiment/from-volume-resume.yaml)
+
+- [Never Resume](./resume-experiment/never-resume.yaml)
+
+## Metrics Collector
+
+Katib supports the various metrics collectors and metrics strategies.
+Check the [official guide](https://www.kubeflow.org/docs/components/katib/experiment/#configuration-spec)
+to know more about it. In this directory you can find the following examples:
+
+- [File Metrics Collector](./metrics-collector/file-metrics-collector.yaml)
+
+- [Custom Metrics Collector](./metrics-collector/custom-metrics-collector.yaml)
+
+- [Metrics Collection Strategy](./metrics-collector/metrics-collection-strategy.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Settings
+
+You can specify different settings for your Trial template. To know more about it
+follow [this guide](https://www.kubeflow.org/docs/components/katib/trial-template/#use-trial-template-to-submit-experiment).
+Check the following examples:
+
+- [Trial with ConfigMap Source](./trial-settings/trial-configmap-source.yaml)
+
+- [Trial with Metadata Substitution](./trial-settings/trial-metadata-substitution.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Training Containers
+
+Check the following examples for Trial training containers:
+
+- [MXNet MNIST](./trial-training-containers/mxnet-mnist)
+
+- [PyTorch MNIST](./trial-training-containers/pytorch-mnist)
```

Comment:
How about including the experiment example manifests using such containers as primary container ?
Highlighting the name of container's image and command is referenced by aboves

Reply:
Sounds good, I added examples in [MXNet README](https://github.com/andreyvelich/katib/tree/refactor-example-folder/examples/v1beta1/trial-training-containers/mxnet-mnist) and [PyTorch README](https://github.com/andreyvelich/katib/tree/refactor-example-folder/examples/v1beta1/trial-training-containers/pytorch-mnist). WDYT ?

Reply:
Looks nice !